.github/workflows/macos-microshift.yml

```diff
@@ -35,6 +35,18 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: ${{ matrix.go }}
+      - name: check if OSP_PULL_SECRET exists
+        shell: bash
+        if: ${{ env.PULL_SECRET == '' }}
+        run: 'echo the secret \"OCP_PULL_SECRET\" has not been made; echo please go to \"settings \> secrets \> actions\" to create it'
```

Comment:
`has not been made` -> `is not set/available/..` ?
This won't stop the execution of the action with a failure though?

Reply:
> This won't stop the execution of the action with a failure though?

yes it won't stop the action. if we want to stop the execution then we need to use `run` with bash script.

Reply:
but the best thing here is this task not run if the pull secret is set.

Reply:
The test is not really useful if it does not stop execution of the whole workflow. I'd remove it if it's only informative